### PR TITLE
Commit focused on renaming to diff tool friendly file names

### PR DESCRIPTION
**Pull request by mwpowellhtx**
Closes the loop on the #331. Thanks!

### CODE REVIEW

**Comment by mwpowellhtx** [comment]
@lperron This is a minimal commit for the ``.mk`` rename, and also includes critical ``clean_third_party`` target fixes. I don't want to get too far ahead of this, but I am actively pursuing unmerged areas.

**Comment by mwpowellhtx** [comment]
@lperron Ran ``third_party`` and ``clean_third_party`` several times to verify. Is working on my end.

**Comment by mwpowellhtx** [comment]
Resolves #330.

**Comment by mwpowellhtx** [comment]
@lperron Just a reminder. Please report what's going on with this PR. Indications are that it can be merged. There are a couple more PRs I want to prioritize before moving forward. Thanks!

File: makefiles/Makefile.third_party.win.mk [hunk 1]
```diff
@@ -17,6 +17,9 @@ SOPLEX_TAG = 2.2.0
 # Version of Sulum
 SULUM_TAG = 43
 
+# Added in support of clean third party targets
+TSVNCACHE_EXE = TSVNCache.exe
```
Comment (lperron) [inline makefiles/Makefile.third_party.win.mk]: I worry if people uses a different svn implementation that does not offer TSVNCACHE

Comment (lperron) [inline makefiles/Makefile.third_party.win.mk]: so I would check if it exists, and then fill in the cleaning macro accordingly.

Alternatively, We add the cleaning macro as a manual step, and you include that in your build process.

**Comment by mwpowellhtx** [comment]
@lperron Yes, I looked into that; that is why the check is in the ``taskkill`` line. If the process isn't there, should be no problem. Please verify, however.

File: makefiles/Makefile.third_party.win.mk [hunk 2]
```diff
@@ -420,9 +423,24 @@ dependencies/install/lib/protobuf.jar: dependencies/install/bin/protoc.exe
...
+# Handle a couple of extraneous circumstances involving TortoiseSVN caching and .svn readonly attributes.
+kill_tortoisesvn_cache:
+	$(TASKKILL) /IM "$(TSVNCACHE_EXE)" /F /FI "STATUS eq RUNNING"
+
+remove_readonly_svn_attribs: kill_tortoisesvn_cache
+	if exist dependencies\sources\* $(ATTRIB) -r /s dependencies\sources\*
+
+
+# Clean everything. Remember to also delete archived dependencies, i.e. in the event of download failure, etc.
+clean_third_party: remove_readonly_svn_attribs
```
Comment (lperron) [inline makefiles/Makefile.third_party.win.mk]: For the time being, can you remove that dependency and have a manual step?

Comment (mwpowellhtx) [inline makefiles/Makefile.third_party.win.mk]: @lperron I found that the clean step FAILS due to permissions issues in the ``.svn`` directory.

File: makefiles/Makefile.third_party.win.mk (hunk 1 above)
Comment (mwpowellhtx) [inline makefiles/Makefile.third_party.win.mk]: @lperron What other version(s) of Subversion would folks be using for Windows?

**Comment by mwpowellhtx** [comment]
@lperron BTW, thanks very much for merging.

**Comment by mwpowellhtx** [comment]
If nothing further other than aforementioned addressed concerns, I will move forward with next goals on my plate.

**Pull request merged**